examples/benchmark/thrift/server.py

```diff
@@ -34,7 +34,7 @@
 
 
 @app.register(KeyValue)
-def getValue(request, response, tchannel):
+def getValue(request, response):
```

Comment:
If we're going to update this example let's switch over to `tchannel.TChannel` instead of `tchannel.tornado.TChannel`.